lib/team.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/player.rb'
+class Team
+  attr_reader :country,
+              :players,
+              :player
```

Comment:
Looks like you're missing a comma here. May throw off the use of the `attr_reader`s.